spec/fixtures/pdf_fill/21-4142-2024/overflow.json

```diff
@@ -30,7 +30,6 @@
       "providerFacilityAddress": {
         "country": "USA",
         "street": "123 Main St",
-        "street2": "A",
```

Comment:
Updated some of our test data to not include street2 to properly test the safety check.